### ISSUE TITLE
Update manual

### ISSUE THREAD

**Issue by sstephenson** [sstephenson/eco#22]
Document the command-line compiler, `eco(...)`, `eco.compile(...)`, templates as Node modules, rendering templates in the browser, and the in-browser compiler.

**Comment by xentek** [comment]
Would love to see how to use this in the browser, especially with precompiled templates. Struggling to piece it all together, and not getting it to output much of anything yet.

**Comment by sstephenson** [comment]
Sorry for the lack of documentation here. I don't have a lot of time to spend on Eco.

There's an example of using the compiler in the browser here: http://sstephenson.github.com/eco/
(dist.js may be a bit out of date...)

**Comment by xentek** [comment]
Yeah I get that. I ended up using handlebars, but will play with eco again (love the erb style better than the moustache style) once I've got a better handle on eco
